test/watch-test.js

```diff
@@ -0,0 +1,290 @@
+/**
+ *
+ *  Web Starter Kit
+ *  Copyright 2016 Google Inc. All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ *
+ */
+
+/* eslint-env node, mocha */
+
+'use strict';
+
+require('chai').should();
+
+const fs = require('fs');
+const path = require('path');
+const ncp = require('ncp');
+const mkdirp = require('mkdirp');
+const rimraf = require('rimraf');
+const taskHelper = require('./helpers/task-helper');
+
+const VALID_TEST_FILES = path.join('test', 'data', 'valid-files');
+const INVALID_TEST_FILES = path.join('test', 'data', 'invalid-files');
+const VALID_TEST_FILES_2 = path.join('test', 'data', 'valid-files-2');
+const TEST_OUTPUT_PATH = path.join('test', 'output');
+const TEST_OUTPUT_SRC = path.join(TEST_OUTPUT_PATH, 'src');
+const TEST_OUTPUT_DEST = path.join(TEST_OUTPUT_PATH, 'build');
+
+let watcherTask;
+
+// Use rimraf over del because it seems to work more reliably on Windows.
+// Probably due to it's retries.
+const deleteFiles = path => {
+  return new Promise((resolve, reject) => {
```

Comment:
What I meant was: Use `path => new Promise(...)` instead of `path => { return new Promise(...); }`